FCCee/FullSim/IDEA/IDEA_o1_v03/run_digi_reco.py

```diff
@@ -97,18 +97,15 @@
     OutputLevel = INFO
 )
 
-# digitize drift chamber hits, waiting to merge the digitizer for the new drift chamber
-#from Configurables import DCHsimpleDigitizerExtendedEdm
-#dch_digitizer = DCHsimpleDigitizerExtendedEdm("DCHsimpleDigitizerExtendedEdm",
-#    inputSimHits = "CDCHHits",
-#    outputDigiHits = "CDCHDigis",
-#    outputSimDigiAssociation = "DC_simDigiAssociation",
-#    readoutName = "CDCHHits",
-#    xyResolution = 0.1, # mm
-#    zResolution = 1, # mm
-#    debugMode = True,
-#    OutputLevel = INFO
-#)
+from Configurables import DCHdigi_v01
+DCHdigi = DCHdigi_v01("DCHdigi")
```

Comment:
Hi Sanghyun, thanks! Not that I have a preference, but since this is the most adopted convention in this file, could you set the parameters of the digitizer with a synthax similar to this:
```
siwrd_digitizer = VTXdigitizer("SiWrDdigitizer",
    inputSimHits = "SiWrDCollection",
    outputDigiHits = "SiWrDDigis",
    outputSimDigiAssociation = "SiWrDSimDigiLinks",
    detectorName = "SiWrD",
    readoutName = "SiWrDCollection",
    xResolution = [siWrapperResolution_x, siWrapperResolution_x], # mm, r-phi direction
    yResolution = [siWrapperResolution_y, siWrapperResolution_y], # mm, z direction
    tResolution = [siWrapperResolution_t, siWrapperResolution_t],
    forceHitsOntoSurface = False,
    OutputLevel = INFO
)

```

Reply:
Done! :)